watson/cli.py

```diff
@@ -68,7 +69,33 @@ def convert(self, value, param, ctx):
             return date
 
 
+class TimeParamType(click.ParamType):
+    name = 'time'
+
+    def convert(self, value, param, ctx):
+        if isinstance(value, arrow.Arrow):
+            return value
+
+        date_pattern = '\d{4}-\d\d-\d\d'  # noqa: W605
+        time_pattern = '\d\d:\d\d(:\d\d)?'  # noqa: W605
```

Comment:
Have you seen this? 
https://lintlyci.github.io/Flake8Rules/rules/W605.html

Reply:
You are right. It should be fixed now.